image/index.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>I.t.t</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <div class="bigContainer">
+      <div class="navbar_container">
+        <span class="img"
+          ><img src="img.webp" alt="logo" class="img1" />
```

Comment:
Provide a proper name for classes, Do not use any numeric value, abbreviations or short forms